migrate_test.go

```diff
@@ -2,12 +2,13 @@ package migrate
 
 import (
 	"database/sql"
+	"fmt"
 	"net/http"
 
+	"github.com/go-gorp/gorp/v3"
 	"github.com/gobuffalo/packr/v2"
 	_ "github.com/mattn/go-sqlite3"
 	. "gopkg.in/check.v1"
-	"gopkg.in/gorp.v1"
```

Comment:
I'm not sure we can do this. Sadly gorp ended up as part of the public API and I guess this will break things for people that depend on it?

Reply:
Mind you, we should probably upgrade, but I don't want to break peoples code if we can avoid it.